Lib/test/test_enum.py

```diff
@@ -2518,6 +2518,7 @@ class Color(StrMixin, AllMixin, Flag):
 
     # TODO: RUSTPYTHON
     @unittest.expectedFailure
+    @unittest.skipIf(sys.platform == "win32", "TODO: RUSTPYTHON, universal newlines")
     @support.reap_threads
```

Comment:
The decorators here are applied in this order (at least in CPython):

1. `support.reap_threads()`
2. `unittest.skipIf()`
3. `unittest.expectedFailure()`